app/(main)/league/[leagueId]/entry/all/page.test.tsx

```diff
@@ -24,6 +24,26 @@ jest.mock('@/api/apiFunctions', () => ({
   ),
 }));
 
+jest.mock('@/components/LeagueEntries/LeagueEntries', () => ({
+  LeagueEntries: ({
+    isPickSet,
+    teamLogo,
+  }: {
+    isPickSet: boolean;
+    teamLogo?: string;
```

Comment:
Why is teamLogo optional here? Would team Logo ever be empty for this application? If that's the case, why are we testing to see if teamLogo exists in lines 129-133?

Reply:
> Why is teamLogo optional here? Would team Logo ever be empty for this application? If that's the case, why are we testing to see if teamLogo exists in lines 129-133?

`teamLogo` is only going to appear when a pick is set. Otherwise it does not exist. That is my thinking as to why it's optional. @vmaineng can of course speak more to it.

Reply:
Correct, @ryandotfurrer . @choir27 please refer to Shashi's comment here as to why it was made optional https://github.com/LetsGetTechnical/gridiron-survivor/pull/376